src/controllers/http-api/v0/local-store-http-api-controller-v0.js

```diff
@@ -65,7 +65,16 @@ class LocalStoreController extends BaseController {
             )}. Operation id: ${operationId}`,
         );
 
-        await this.operationIdService.cacheOperationIdData(operationId, cachedAssertions);
+        try {
+            await this.operationIdService.cacheOperationIdData(operationId, cachedAssertions);
+        } catch (err) {
+            this.operationIdService.updateOperationIdStatus(
+                operationId,
+                null,
+                OPERATION_ID_STATUS.FAILED,
```

Comment:
Use error message and error type. Error message should be extracted from err and type should be read from constants. See PUBLISH_ROUTE_ERROR for reference